io.vikunja.Desktop.yml

```diff
@@ -0,0 +1,42 @@
+app-id: io.vikunja.Desktop
+app-version: '0.21.0'
+version-hash: '46bd28e13d35f78bbcf2d3b44307059b195f5d1eefcd08937645888f008365e3'
+
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+
+build-options:
+  append-path: /usr/lib/sdk/node18/bin
+command: vikunja-desktop
+
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications
+  - --env=LD_LIBRARY_PATH=/app/lib
+  - --env=ZYPAK_SANDBOX_FILENAME=chrome-sandbox
+  
+modules:
+  - name: vikunja
+    buildsystem: simple
+    build-commands:
+      - mkdir /app/bin/extra /app/lib/extra
+      - mv *.so* /app/lib/
+      - install -D vikunja.sh "/${FLATPAK_DEST}/bin/vikunja"
+      - mv * /app/bin/extra/
```

Comment:
I don't see an icon or a .desktop file installed.

Reply:
They are in the tarball.

Reply:
then they are not installed in the right places.

Reply:
This is still unresolved (but you resolved the thread). Its not a "nice to have".